gemma-teixido/pre-curso/tema-1/calculator-pro.js

```diff
@@ -0,0 +1,114 @@
+let resultArr = [];
+
+function message() {
+  let messageNewNumber = prompt("New numbers? y/n");
+  switch (messageNewNumber) {
+    case "n":
+      let stop = "Bye!";
+      console.log(stop);
+      return stop;
+
+    case "y":
+      let numbers = prompt("Introduce numbers separated by commas");
+      let numberList = numbers.split(",");
+      for (let i = 0; i < numberList.length; i++) {
```

Comment:
Declare numbers and numberList at the top. Then assign here to numbers and numerList the split and the prompts.